extensions/spark/kyuubi-spark-authz/src/main/scala/org/apache/kyuubi/plugin/spark/authz/PrivilegeObject.scala

```diff
@@ -43,4 +43,5 @@ case class PrivilegeObject(
     actionType: PrivilegeObjectActionType,
     dbname: String,
     objectName: String,
-    @Nonnull columns: Seq[String] = Nil)
+    @Nonnull columns: Seq[String] = Nil,
+    owner: Option[String] = None)
```

Comment:
`PrivilegeObject` is identifying types and dimensions of what privilege to be checked. This `owner` added here in the constructor is more like extra information about the resource itself rather than the description of privilege. I think it might violate the purpose of this `PrivilegeObject`. Is there any better we could defer it in later privilege checking in RuleAuthorization? Or we just have a flexible map here for now and future extended resource info like maybe `ownerGroup`, `resourceTag` or etc.?

Reply:
`PrivilegeObject` originates from `HivePrivilegeObject`. In `HivePrivilegeObject`, owner is stored in a field named `ownerName` as a sibling to `dbname` and `columns`.
I'd like to keep the code as it is for now to follow Hive's design.  We can optimize the code if things get complicated in the future.

Reply:
OK, good to know it. 
Let me attach refererence link to `ownerName` field of `HivePrivilegeObject` from hive-exec:
https://github.com/apache/hive/blob/master/ql/src/java/org/apache/hadoop/hive/ql/security/authorization/plugin/HivePrivilegeObject.java#L134